src/agent/README.md

```diff
@@ -39,11 +39,21 @@ After that, we drafted the initial code here, and any contributions are welcome.
 ## Getting Started
 
 ### Build from Source
-The rust-agent need to be built with rust newer than 1.37, and static linked with `musl`.
+The rust-agent need to be built with static linked with musl:
 ```bash
-rustup target add x86_64-unknown-linux-musl
-sudo ln -s /usr/bin/g++ /bin/musl-g++  
-cargo build --target x86_64-unknown-linux-musl --release
+$ arch=$(uname -m)
+$ [ "$arch" == "ppc64le" ] && arch=powerpc64le
+$ rustup target add "${arch}-unknown-linux-musl"
+$ sudo ln -s /usr/bin/g++ /bin/musl-g++
+```
+Download the kata-containers source and build the agent:
```

Comment:
Please can you change "kata-containers" to "Kata Containers".

Reply:
@jodh-intel I disagree, this refers to the repository name. I would instead replace it with `kata-containers` using back-ticks.